### PR TITLE
Scrum10/build add button

### PR DESCRIPTION
**Pull request by Omarmanassrah**
The "Add" button allows users to dynamically create a new card on the page. When clicked, a new card element is appended to the container, which includes a placeholder text that the user can edit. Each new card can be customized with different background colors, edited, or deleted. Additionally, the card includes a timestamp showing the date when it was added.

### CODE REVIEW

**Review by Omarjabari007 — APPROVED** [review]
Update the pr.

**Review by Omarjabari007 — COMMENTED** [review]
51 files changed?

only add the required code.

**Review by Omarjabari007 — APPROVED** [review]

**Review by Omarjabari007 — APPROVED** [review]

File: index.html [hunk 1]
```diff
@@ -3,9 +3,78 @@
...
+    <!-- Header Section (from dev branch) -->
+    <header class="header">
+      <nav class="nav">
+        <ul class="UnOrderdBoardList">
+          <li class="BoardList"><a href="">Board 1</a></li>
+          <li class="BoardList"><a href="">Board 2</a></li>
+        </ul>
+      </nav>  
+      <div class="right" id="right">
+        <input type="text" placeholder="   &#x1F50D;search a note" id="search" class="search" />
```
Comment (kldoon) [inline index.html:19]: you can use padding rather than the empty spaces

File: index.html [hunk 2]
```diff
@@ -3,9 +3,78 @@
...
     <link rel="stylesheet" href="style.css" />
-    <script defer src="script.js"></script>
+    <title>Card</title>
   </head>
-  <body></body>
+  <body>
+    <!-- Header Section (from dev branch) -->
+    <header class="header">
+      <nav class="nav">
+        <ul class="UnOrderdBoardList">
```
Comment (kldoon) [inline index.html:13]: we should use unified class names convention

File: script.js [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+    const textarea = document.createElement("textarea");
+    textarea.value = textContent.textContent.trim();
+    textarea.style.width = "100%";
+    textarea.style.height = "100px";
+    textarea.style.marginBottom = "20px";
+
+    card.replaceChild(textarea, textContent);
+
+    const saveButton = document.createElement("button");
+    saveButton.textContent = "Save";
```
Comment (kldoon) [inline script.js:64]: it's better to have the styles in the css file

**Pull request merged**